.github/workflows/release-bumper.yml

```diff
@@ -99,4 +94,4 @@ jobs:
           assignees: tiraboschi,orenc1,nunnatsa
           reviewers: tiraboschi,orenc1,nunnatsa
           team-reviewers: owners, maintainers
-          branch: bump_${{ env.UPDATED_COMPONENT }}_${{ env.UPDATED_VERSION }}
+          branch: bump_${{ env.UPDATED_COMPONENT }}_${{ env.UPDATED_VERSION }}_${{ matrix.branches.branch }}
```

Comment:
Because of the problem described in [this comment](https://github.com/kubevirt/hyperconverged-cluster-operator/pull/1738/files#r790982303), multiple stabilization branches were getting an update to the same versions of a component.

When that happened, since the name of the branch with the changes only contains the name and version of the updated component, it happened that the same branch name was being used for multiple stabilization branches.

This, in turn, caused the last stabilization branch to be updated in the workflow to override the update branch with the commits from the stabilization branch. That affects the previously opened PRs for other stabilization branches that were trying to merge the exact same branch, that now would have different commits.